o!mAC: Initial article (#2594)
ppy/osu-wiki: wiki/Tournaments/o!mAC/2019/en.md

# osu!mania Asian's Champions 2019

![o!mAC2019 Logo](img/logo.png)

The **osu!mania Asian's Champions 2019** (***o!mAC2019***) is an individual 4K mania tournament intended for all osu!mania players across the Asia Continent.

## Tournament schedule

| Event | Timestamp |
| --: | :-- |
| Registration Phase | 2019-08-11/2019-08-31 |
| Qualifiers | 2019-09-21/2019-09-22 |
| Live Drawings | 2019-09-23 21:00 (UTC+7) |
| Group Stage | 2019-09-28/2019-10-06 |
| Round of 16 | 2019-10-12/2019-10-13 |
| Quarterfinals | 2019-10-19/2019-10-20 |
| Semifinals | 2019-10-26/2019-10-27 |
| Finals - Week 1 | 2019-11-02/2019-11-03 |
| Finals - Week 2 | 2019-11-09/2019-11-10 |

## Prizes

| Placing | Prize(s) |
| :-: | :-- |

> | ![Gold Crown](/wiki/shared/GCrown.png "1st place") | Profile Badge (To be confirmed), 4 month of osu!supporter tag, userpage banner |

Reviewer: ```suggestion
| ![Gold Crown](/wiki/shared/GCrown.png "1st place") | Profile badge (pending confirmation), 4 months of osu!supporter tag, userpage banner |
```

> | ![Silver Crown](/wiki/shared/SCrown.png "2nd place") | 3 month of osu!supporter tag, userpage banner |

Reviewer: ```suggestion
| ![Silver Crown](/wiki/shared/SCrown.png "2nd place") | 3 months of osu!supporter tag, userpage banner |
```

> | ![Bronze Crown](/wiki/shared/BCrown.png "3rd place") | 2 month of osu!supporter tag, userpage banner |

Reviewer: ```suggestion
| ![Bronze Crown](/wiki/shared/BCrown.png "3rd place") | 2 months of osu!supporter tag, userpage banner |
```

## Organisation

> The osu!mania Asian's Champions 2019 is run by various community members by distributing the multitude of tasks into various fields of responsibility.

Reviewer: ```suggestion
The osu!mania Asian's Champions 2019 is run by various community members.
```

| Position | Member(s) |
| :-- | :-- |
| Host | ![][flag_ID] [chouyaa](https://osu.ppy.sh/users/8404646) |
| Co-host | ![][flag_PH] [vincent4399](https://osu.ppy.sh/users/9764388) |
| Map Selector | ![][flag_ID] [Virtue-](https://osu.ppy.sh/users/6144772), ![][flag_TH] [RuleBlazing](https://osu.ppy.sh/users/7312402), ![][flag_SG] [riunosk](https://osu.ppy.sh/users/5594381) |
| Streamer | ![][flag_DE] [DinX](https://osu.ppy.sh/users/568111), ![][flag_NL] [NightNarumi](https://osu.ppy.sh/users/4381142), ![][flag_LV] [Adiopulse](https://osu.ppy.sh/users/4387852) |
| Commentator | ![][flag_VN] [Weabonie](https://osu.ppy.sh/users/8151986), ![][flag_TH] [HowToPlayLN](https://osu.ppy.sh/users/10879600) |
| Referee | ![][flag_TH] [TNKSTC](https://osu.ppy.sh/users/6037121), ![][flag_ID] [Noch Einen](https://osu.ppy.sh/users/12668735), ![][flag_ID] [FadillSan](https://osu.ppy.sh/users/7006830), ![][flag_ID] [RiP46](https://osu.ppy.sh/users/5222596), ![][flag_TW] [Makura](https://osu.ppy.sh/users/10665623), ![][flag_FR] [Mathisca](https://osu.ppy.sh/users/2227462) |
| Statistician | ![][flag_TH] [mashedpotato_old](https://osu.ppy.sh/users/496237) |

> | Recap Editor | ![][flag_KR] [Misuku](https://osu.ppy.sh/users/4799593) |

Reviewer: ```suggestion
| Recap editor | ![][flag_KR] [Misuku](https://osu.ppy.sh/users/4799593) |
```

> | Wiki | ![][flag_CN] [kj415j45](https://osu.ppy.sh/users/9367540) |

Reviewer: ```suggestion
| Wiki editor | ![][flag_CN] [kj415j45](https://osu.ppy.sh/users/9367540) |
```

## Links

- [Discussion thread](https://osu.ppy.sh/community/forums/topics/946117)
- [Challonge](https://challonge.com/zh_CN/asianschampions2019)
- [Livestream](https://www.twitch.tv/asianschampions)

> - **Statistics sheet**

Reviewer: ```suggestion
```

>   - [Mappool](https://docs.google.com/spreadsheets/d/1aGQHz8YRl-8a8mdp-7E5h_KQBk3rnwlvUzePaIrx6Zs)

Reviewer: ```suggestion
- [Mappool spreadsheet](https://docs.google.com/spreadsheets/d/1aGQHz8YRl-8a8mdp-7E5h_KQBk3rnwlvUzePaIrx6Zs)
```

>   - [Schedule](https://docs.google.com/spreadsheets/d/1vbSfQXcCG-CMeBmwlCKN5WW1gaaY48yqGsjyFvcN0zU)

Reviewer: ```suggestion
- [Schedule spreadsheet](https://docs.google.com/spreadsheets/d/1vbSfQXcCG-CMeBmwlCKN5WW1gaaY48yqGsjyFvcN0zU)
```

## Mappools

### Group Stage

- FreeMod

>   - [Tennyson - No Answer \[Silence\]](https://osu.ppy.sh/beatmapsets/977509#mania/2045881)

Reviewer: map format should go `Artist - Title (Creator) [Diff]`, & don't write Source before this

  - [Thaehan - Doki-Doki \[4k Hard\]](https://osu.ppy.sh/beatmapsets/579878#mania/1227947)
  - [ZUN - Plastic Mind.arr \[Lvl 15\]](https://osu.ppy.sh/beatmapsets/485425#mania/1035232)
  - [Shawn Wasabi - Pizza Rolls (Metal x EDM Mashup) \[Adtw's WTF\]](https://osu.ppy.sh/beatmapsets/333704#mania/741543)
  - [brothel. - timmy. \[notitle.\]](https://osu.ppy.sh/beatmapsets/697911#mania/1478256)
- Tiebreaker
  - **[Team Grimoire - MOTHER FUCXXXXXer \[4K nightmare Lady\]](https://osu.ppy.sh/beatmapsets/568931#mania/1206166)**

### Qualifiers

- [Culprate & Dictate - Pencilina \[Stage 1 : Paranoid\]](https://osu.ppy.sh/beatmapsets/991059#mania/2072838)
- [Hyper Potions - Ski \[Stage 2: Lubricity\]](https://osu.ppy.sh/beatmapsets/1030043#mania/2153826)
- [senya - Kimi to Dareka no Yasashisa ni \[Stage 3 : Great Harmony\]](https://osu.ppy.sh/beatmapsets/981019#mania/2053009)
- [SOUND VOLTEX IV HEAVENLY HAVEN (Laur) - Absolute Domination \[Stage 4 : Devastation\]](https://osu.ppy.sh/beatmapsets/1031176#mania/2156349)
- [東方紅魔郷 ~ the Embodiment of Scarlet Devil. (PHOENIX Project) - Youma Yakou \[Stage 5: Umbrakinesis\]](https://osu.ppy.sh/beatmapsets/1012614#mania/2119478)

## Ruleset

> ### Tournament Registration

Reviewer: ```suggestion
### Tournament registration
```

- The flag on your profile must be inside one of Asian continent. For complete list you can visit [following page](https://www.worldometers.info/geography/how-many-countries-in-asia/)

> ### Match Instructions

Reviewer: ```suggestion
### Match instructions
```

> - You will get a notification from your referee that your match is ready 10 minutes before the schedule. Once you are ready, please let them know and they will invite you to the lobby soonly.

Reviewer: ```suggestion
- You will get a notification from your referee that your match is ready 10 minutes before the scheduled time. Once you are ready, please let them know, and they will invite you to the lobby.
```

> - The referee will ask you to roll for map banning and the first pick. The highest roll goes first for map banning, then the next player goes afterwards. This applies vice versa on the first map picker.

Reviewer: ```suggestion
- The referee will ask you to roll for ban and pick order. The highest roll will ban first, and the lowest roll will pick first. Ban and pick order alternates between players.
```

- Warmup maps must be downloadable via osu!direct with 3 minutes length at maximum.

> - Visual mods such as HD, FL, FI are allowed.

Reviewer: ```suggestion
- Visual mods such as HD, FL, FI and MR are allowed.
```

> - After the warmups, each players will have 1 ban of any map.

Reviewer: ```suggestion
- After the warmups, each player will ban one map.
```

> - Players will pick maps alternating with each other until one player reach the required amount of points to win the match.

Reviewer: ```suggestion
- Players will pick maps until one player reaches the required amount of points to win the match.
```

> - Tiebreaker map will be played if both players are at match point.

Reviewer: ```suggestion
- The tiebreaker map will be played if both players are at match point.
```

# osu!mania Asian's Champions 2019

![o!mAC2019 Logo](img/logo.png)

> The **osu!mania Asian's Champions 2019** (***o!mAC2019***) is an individual 4K mania tournament intended for all osu!mania players in Asia continental.

Reviewer 2: ```suggestion
The **osu!mania Asian's Champions 2019** (***o!mAC2019***) is an individual 4K osu!mania tournament intended for all osu!mania players across the Asia Continent.
```

> ## Tournament Schedule

Reviewer 2: ```suggestion
## Tournament schedule
```

| Event | Timestamp |
| --: | :-- |
| Registration Phase | 2019-08-11/2019-08-31 |
| Qualifiers | 2019-09-21/2019-09-22 |
| Live Drawings | TBD |
| Group Stage | 2019-09-28/2019-10-06 |
| Round of 16 | 2019-10-12/2019-10-13 |
| Quarterfinals | 2019-10-19/2019-10-20 |
| Semifinals | 2019-10-26/2019-10-27 |
| Finals - Week 1 | 2019-11-02/2019-11-03 |
| Finals - Week 2 | 2019-11-09/2019-11-10 |

## Prizes

> In every tournament, people conquering a place on the podium are eligible to receive unique prizes created specifically for osu!mania Asian's Champions winners. It is possible that these items change with every installment of the o!mAC.

Reviewer 2: We can remove this. was treated as just piece of text with no purpose.

| Placing | Prize(s) |
| :-: | :-- |

> | ![Gold Crown](/wiki/shared/GCrown.png "1st place") | Profile Badge (To be confirmed), 1 month supporter tag, userpage banner |

Reviewer 2: ```suggestion
| ![Gold Crown](/wiki/shared/GCrown.png "1st place") | Profile Badge (To be confirmed), 1 month of  osu!supporter tag, userpage banner |
```
Maybe makes that better to read?

> | ![Silver Crown](/wiki/shared/SCrown.png "2nd place") | 1 month supporter tag, userpage banner |

Reviewer 2: ```suggestion
| ![Silver Crown](/wiki/shared/SCrown.png "2nd place") | 1 month of osu!supporter tag, userpage banner |
```

> | ![Bronze Crown](/wiki/shared/BCrown.png "3rd place") | userpage banner |

Reviewer 2: ```suggestion
| ![Bronze Crown](/wiki/shared/BCrown.png "3rd place") | Userpage banner |
```

| Statistician | ![][flag_TH] [mashedpotato_old](https://osu.ppy.sh/users/496237) |
| Wiki | ![][flag_CN] [kj415j45](https://osu.ppy.sh/users/9367540) |

## Links

- [Discussion thread](https://osu.ppy.sh/community/forums/topics/946117)
- **Statistics sheet**
  - [Mappool](https://docs.google.com/spreadsheets/d/1aGQHz8YRl-8a8mdp-7E5h_KQBk3rnwlvUzePaIrx6Zs)
  - [Schedule](https://docs.google.com/spreadsheets/d/1vbSfQXcCG-CMeBmwlCKN5WW1gaaY48yqGsjyFvcN0zU)

## Mappools

### Qualifiers

- [Culprate & Dictate - Pencilina \[Stage 1 : Paranoid\]](https://osu.ppy.sh/beatmapsets/991059#mania/2072838)
- [Hyper Potions - Ski \[Stage 2: Lubricity\]](https://osu.ppy.sh/beatmapsets/1030043#mania/2153826)
- [senya - Kimi to Dareka no Yasashisa ni \[Stage 3 : Great Harmony\]](https://osu.ppy.sh/beatmapsets/981019#mania/2053009)
- [SOUND VOLTEX IV HEAVENLY HAVEN (Laur) - Absolute Domination \[Stage 4 : Devastation\]](https://osu.ppy.sh/beatmapsets/1031176#mania/2156349)
- [東方紅魔郷 ~ the Embodiment of Scarlet Devil. (PHOENIX Project) - Youma Yakou \[Stage 5: Umbrakinesis\]](https://osu.ppy.sh/beatmapsets/1012614#mania/2119478)

## Ruleset

### Tournament Registration

> - You must be in Asia continent countries

Reviewer 2: ```suggestion
- The flag on your profile must be inside one of Asian continent. For complete list you can visit [following page](https://www.worldometers.info/geography/how-many-countries-in-asia/)
```
I'd prefer to specify with the countries that can participate, you can link it to other sites instead of putting all country in the wiki.

### Match Instructions

- You will get a notification from your referee that your match is ready 10 minutes before the schedule. Once you are ready, please let them know and they will invite you to the lobby soonly.
- The referee will ask you to roll for map banning and the first pick. The highest roll goes first for map banning, then the next player goes afterwards. This applies vice versa on the first map picker.
- Warmup maps must be downloadable via osu!direct with 3 minutes length at maximum.
- Visual mods such as HD, FL, FI are allowed.
- After the warmups, each players will have 1 ban of any map.
- Players will pick maps alternating with each other until one player reach the required amount of points to win the match.

> - Tie-breaker map will be played if both players are at match point.

Reviewer 2: ```suggestion
- Tiebreaker map will be played if both players are at match point.
```
I think we agreed to have it without dash some time ago.
